_content/home-page.md

```diff
@@ -1,38 +1,39 @@
 ---
 last_updated_time: 2021-07-24T05:25:22.878Z
 ---
-*\#WargaBantuWarga adalah situs yang memuat informasi seputar COVID-19 dan kontak fasilitas/alat kesehatan di seluruh Indonesia yang dikumpulkan relawan melalui pencarian di internet atau media sosial.*
 
-- - -
+_\#WargaBantuWarga adalah situs yang memuat informasi seputar COVID-19 dan kontak fasilitas/alat kesehatan di seluruh Indonesia yang dikumpulkan relawan melalui pencarian di internet atau media sosial._
+
+---
```

Comment:
Just FYI @ekafyi, these formatting changes in this *.md file will eventually be overwritten by Netlify CMS. :sweat_smile:

Reply:
@zainfathoni We should probably make prettier + markdownlint ignore the entire `_content` folder while we're at it.